packages/car_interface_dbv2/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <name>car_interface_dbv2</name>
+  <version>1.0.0</version>
+  <description>The car_interface meta package</description>
+
+  <license>GPLv3</license>
+<maintainer email="mack@duckietown.org">Mack</maintainer>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <!-- <build_depend>duckietown_msgs</build_depend> -->
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+
+  <!-- <run_depend>duckietown_msgs</run_depend> -->
+  <run_depend>roscpp</run_depend>
+  <run_depend>rospy</run_depend>
```

Comment:
same comment, doesnt it need cpp, also replace run and build_depend by just depend